.github/workflows/databend-release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Rust setup
         run: |
           bash ./scripts/setup/dev_setup.sh
-          cargo install --version 0.2.1 cross
+          cargo install cross
```

Comment:
I think we should keep exact version number to get avoid of surprising breaking change

Reply:
```suggestion
          cargo install --version 0.2.1 cross
```

Reply:
Makes sense, addressed.